### ISSUE TITLE
SPM Package Resolution Failed.

### ISSUE THREAD

**Issue by JulesMoorhouse** [hackiftekhar/IQKeyboardManager#1859]
**Describe the bug**
When trying to add this repo a swift package... 
```
https://github.com/hackiftekhar/IQKeyboardManager.git
```
...the follow error is produced...
```
/Package.swift has no Package.swift manifest for version 2.5.0 in https://github.com/hackiftekhar/IQKeyboardManager.git
```

**To Reproduce**
Add the package via url, add and see a package could not be resolved dialog

**Expected behavior**
No dialog to be shown

**Screenshots**
<img width="656" alt="Untitled" src="https://user-images.githubusercontent.com/361024/150848164-855080e8-fb87-41f6-8708-062a609cc8e1.png">

### Versions
**Xcode:** 13.2.1
**Mac OS:** 11.6.2
**Simulator/Device:** iOS 14.5
**Simulator/Device Name:** iPhone 8
**Library Version:** 2.5.0

**Comment by emtjoshhart** [comment]
Getting this too.  Package.swift not included in latest version.  Could it be added?

**Comment by ChrisSherlock91** [comment]
Still seeing this issue even with updating the version to the latest also tried historical releases with no luck. Any further updates on this issue?

**Comment by yycking** [comment]
Hi @ChrisSherlock91 
You must remove it from package dependencies and add it again.

**Comment by sh1j1th** [comment]
@ChrisSherlock91, Instead of setting a specific version, choosing master branch in the version rules worked for me. Initially, I had the **Package resolution failed** error but when I made an attempt to reload, the version rule was auto-set to master branch and re-adding resolved the issue. I deleted the other version too before re-building.

<img width="703" alt="Screenshot 2022-04-05 at 11 22 06 PM" src="https://user-images.githubusercontent.com/38531459/161818963-c0b8e389-3e74-484d-a347-a4166fad7f9f.png">
